code/datums/status_effects/debuffs.dm

```diff
@@ -16,6 +16,10 @@
 	if(update_canmove)
 		owner.update_canmove()
 
+//STUN
+/datum/status_effect/incapacitating/stun
+	id = "stun"
```

Comment:
wow it's like two fucking lines to make a status effect now, all you need to do is add checks where you need them, isn't that wonderful